README.md

```diff
@@ -336,8 +381,44 @@ There are a few points to be made here:
   run, we ignore that field with the `ignore` property.
 
 
+### Version 2.0 (Daemon mode only)
+
+With version 2.0 of Logstash Filter Verifier (Daemon mode) some new features
+have been added:
+
+* **Export of @metadata**:  
+  There is out of the box support to let Logstash Filter Verifier export
+  the values in the (otherwise hidden) `@metadata` field of the event.
+  This allows to write test cases, which take the values in the `@metadata`
+  field into account.
+* **Pipeline configuration**:  
+  Logstash Filter Verifier in Daemon mode accepts complete Logstash pipelines
+  as configuration. This includes the localization of the Logstash configuration
+  files through the paths provided in the `pipelines.yml` file and replacing all
+  input and output filters with the respective parts to execute the tests.
+* **Multiple pipelines**  
+  The pipeline configuration may consist of multiple pipelines, that might be
+  linked ([pipeline to pipeline communication](https://www.elastic.co/guide/en/logstash/current/pipeline-to-pipeline.html))
+  or independent pipelines.
+* **Filter mock**  
+  Filter mock allows to replace (or remove) filter plugins in the Logstash
+  configuration under test, that do not work during or that would potentially
+  not produce the expected results test execution. Examples for such filter
+  plugins are mainly plugins, that perform some sort of call out to a third
+  party system, for example to look up data ([elasticsearch](https://www.elastic.co/guide/en/logstash/current/plugins-filters-elasticsearch.html),
+  [http](https://www.elastic.co/guide/en/logstash/current/plugins-filters-http.html),
+  [jdbc](https://www.elastic.co/guide/en/logstash/current/plugins-filters-jdbc_static.html),
+  [memcached](https://www.elastic.co/guide/en/logstash/current/plugins-filters-memcached.html)).
+  In order to to be able to produce reproducible results in the test cases,
+  these plugins can be replaced with mocks. In particular the [mutate](https://www.elastic.co/guide/en/logstash/current/plugins-filters-mutate.html)
+  and the [translate](https://www.elastic.co/guide/en/logstash/current/plugins-filters-translate.html)
+  filters have proven to be helpful as replacements.
```

Comment:
This bullet list of features introduced in 2.0 is in the _Examples_ section yet no examples are given :-) I think this text is in the right place but it should be complemented with actual examples. Perhaps it would be a good idea if I provided the examples in a follow-up PR? That would be a good excuse for me to really learn all of it. Although I've looked at your PRs I can't say I know them be heart and I'll probably have to look in the source code to figure out how to use some of them.

Reply:
You are right, I added some instructions on how to execute a test case from the `testcases` folder (the cases I use in the integration tests). I hope, this will get you started. Feel free to extend this section with more examples.